src/styles/blocks/header.scss

```diff
@@ -0,0 +1,63 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  height: 60px;
+  &__icon {
+    width: 40px;
+    height: 40px;
+    img {
+      display: block;
+    }
+  }
+
+  &__list {
+    display: flex;
+    align-items: center;
+    font-weight: 500;
+    padding: 0;
+    margin: 0;
+  }
+  &__item {
+    list-style: none;
+    margin-right: 20px;
+    text-decoration: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+  }
```

Comment:
The `text-decoration: none;` style is not needed here as it doesn't apply to list items. You can move it to the `&__item a` selector where it's more relevant.